pkg/vmimage/data/script.sh

```diff
@@ -178,6 +178,8 @@ rm /etc/yum.repos.d/kickstart.repo /var/lib/yum/client-cert.pem /var/lib/yum/cli
 
 mkdir /var/lib/etcd
 
+mkdir -m 0750 /var/lib/origin/openshift.local.volumes
```

Comment:
comment here about quota code needing this would be useful

Reply:
ironic right? I ended up following the comment style from above and below just in case it would be deemed too obvious. I need to revisit this anyway since there a go panic at the moment when I run "make vmimage"